### PR TITLE
Fix issue that prevented sending stats when openvpn client disconnect

### PR DESCRIPTION
**Pull request by fernando-guerrero-leon**
Added absolute path for IPSEC and IPSET

Avoids sending stats of recently disconnected client

New function "_get_openvpn_disconnected_clients" to know the common_name of the disconnected client

Error handling for VPN_SESSION

### CODE REVIEW

File: encryptme_stats/metrics.py [hunk 1]
```diff
@@ -583,14 +605,18 @@ def vpn_session():
...
-        return empty
+        result = openvpn_stat + ipsec_stat
+        if len(result) == 0:
+            return empty
 
-    return result
+        return result
+    except Exception as exc:
+        logging.debug("Error gathering vpn_session stats: %s", exc)
+        return {}
```
Comment (dmibaranov) [inline encryptme_stats/metrics.py]: `return empty`

File: encryptme_stats/metrics.py [hunk 2]
```diff
@@ -583,14 +605,18 @@ def vpn_session():
 
     :return: list of dictionaries with connections statistics
     """
-    empty = []
-    return empty  # disable vpn_session metrcis for now
+    try:
+        empty = []
+        return empty  # disable vpn_session metrcis for now
```
Comment (dmibaranov) [inline encryptme_stats/metrics.py]: please enable sending stats and remove return here

File: encryptme_stats/metrics.py [hunk 3]
```diff
@@ -583,14 +605,18 @@ def vpn_session():
 
     :return: list of dictionaries with connections statistics
     """
-    empty = []
-    return empty  # disable vpn_session metrcis for now
+    try:
+        empty = []
```
Comment (dmibaranov) [inline encryptme_stats/metrics.py]: should outside of try block

File: encryptme_stats/metrics.py (hunk 1 above)
Comment (fernando-guerrero-leon) [reply encryptme_stats/metrics.py]: Done in new commit

File: encryptme_stats/metrics.py (hunk 2 above)
Comment (fernando-guerrero-leon) [reply encryptme_stats/metrics.py]: Done

File: encryptme_stats/metrics.py (hunk 3 above)
Comment (fernando-guerrero-leon) [reply encryptme_stats/metrics.py]: Done

**Pull request merged**